modules/ROOT/partials/configuration/revisionhistory_fetch_revision.adoc

```diff
@@ -0,0 +1,87 @@
+[[revisionhistory_fetch_revision]]
+== `revisionhistory_fetch_revision`
+
+The plugin calls this function to fetch a revision's content when a revision is selected in the revision history dialog. This function is called with the revision object that was selected in the revision history dialog. The function should return a Promise that resolves with the revision content.
+
+*Type:* `+Function+` (Returns a https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Promise[Promise])
+
+*Input parameters:*
+[cols="1,1,1,3",options="header"]
+|===
+|Field |Type |Required? |Description
+|`+editor+` | `+Editor+` | required | The editor instance to load the revision into (useful if there are multiple editors on the page).
+|`+revision+` | `+Object+` | required | The revision object to load (must include `revisionId` field).
+|===
+
+*Return data:* revision `+Object+` with the following fields:
+[cols="1,1,3",options="header"]
+|===
+|Field |Type |Required? |Description
+|`+revisionId+` | `+string+` | required | The unique string Id of the revision.
+|`+createdAt+` | `+string+` | required | A UTC datetime string in ISO-8061 format.
+|`+content+` | `+string+` | required | HTML string of the revision content.
+|===
```

Comment:
Can this be a link to the revision object defined in `revisionhistory_fetch`?

Reply:
[f6a8f86](https://github.com/tinymce/tinymce-docs/pull/3349/commits/f6a8f86a59791e76f9c2cd3be143dba38c3170c6)